package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.36.0",
+  "version": "3.36.1",
```

Comment:
Don't know how much it matters at this point, but it might be worth bumping the minor version instead of just the patch. :man_shrugging:

Reply:
Yeah, I flip-flopped on this (hah, look at the commit history), but your opinion makes me second-guess myself. I'll change it to a minor release.